x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/flyout_validations.cy.ts

```diff
@@ -109,235 +109,256 @@ describe('Exceptions flyout', { testIsolation: false }, () => {
-      .eq(3)
-      .should('have.text', '@timestamp');
-
-    // deleting the last value of a nested entry, should delete the child and parent
-    cy.get(ENTRY_DELETE_BTN).eq(4).click();
-    cy.get(EXCEPTION_ITEM_CONTAINER)
-      .eq(0)
-      .find(FIELD_INPUT_PARENT)
-      .eq(0)
-      .should('have.text', 'agent.name');
-    cy.get(EXCEPTION_ITEM_CONTAINER).eq(0).find(FIELD_INPUT_PARENT).eq(1).should('have.text', 'b');
-    cy.get(EXCEPTION_ITEM_CONTAINER)
-      .eq(1)
-      .find(FIELD_INPUT_PARENT)
-      .eq(0)
-      .should('have.text', 'agent.name');
-    cy.get(EXCEPTION_ITEM_CONTAINER)
-      .eq(1)
-      .find(FIELD_INPUT_PARENT)
-      .eq(1)
-      .should('have.text', '@timestamp');
-  });
+      // add an entry with a value and submit button should enable
+      addExceptionEntryFieldValue('agent.type', 0);
+      cy.get(VALUES_INPUT).eq(0).type(`{enter}`);
+      cy.get(VALUES_INPUT).eq(0).type(`{downarrow}{enter}`);
+      cy.get(CONFIRM_BTN).should('be.enabled');
+    });
 
-  it('Contains custom index fields', () => {
-    // open add exception modal
-    openExceptionFlyoutFromEmptyViewerPrompt();
+    it('Validates empty entry values correctly', () => {
+      // open add exception modal
+      openExceptionFlyoutFromEmptyViewerPrompt();
 
-    searchExceptionEntryFieldWithPrefix('unique');
-    cy.get(EXCEPTION_FIELD_LIST).contains('unique_value.test');
-  });
+      // add exception item name
+      addExceptionFlyoutItemName('My item name');
 
-  it('Validates auto-suggested fields correctly', () => {
-    // open add exception modal
-    openExceptionFlyoutFromEmptyViewerPrompt();
+      // add an entry with a value and submit button should enable
+      addExceptionEntryFieldValue('agent.name', 0);
+      addExceptionEntryFieldValueValue('test', 0);
+      cy.get(CONFIRM_BTN).should('be.enabled');
 
-    // add exception item name
-    addExceptionFlyoutItemName('My item name');
+      // remove the value and should see warning and button disabled
+      cy.get(VALUES_INPUT).eq(0).type('{backspace}{enter}');
+      cy.get(EXCEPTION_FLYOUT_TITLE).click();
+      cy.get(CONFIRM_BTN).should('be.disabled');
 
-    // add an entry with a value and submit button should enable
-    addExceptionEntryFieldValue('agent.type', 0);
-    cy.get(VALUES_INPUT).eq(0).type(`{enter}`);
-    cy.get(VALUES_INPUT).eq(0).type(`{downarrow}{enter}`);
-    cy.get(CONFIRM_BTN).should('be.enabled');
+      // change operators
+      addExceptionEntryOperatorValue('is one of', 0);
+      cy.get(CONFIRM_BTN).should('be.disabled');
+
+      // add value again and button should be enabled again
+      addExceptionEntryFieldMatchAnyValue('test', 0);
+      cy.get(CONFIRM_BTN).should('be.enabled');
+    });
   });
 
-  it.skip('Warns users about mapping conflicts on problematic field selection', async () => {
-    // open add exception modal
-    openExceptionFlyoutFromEmptyViewerPrompt();
+  // TODO: figure out why it's skipped
```

Comment:
We disabled it here https://github.com/elastic/kibana/pull/152962. For some reason this test is not stable and fails often.

Might be related to:

```
I think this might be related to other issues where the exception flyout opens and then cypress loses the input box focus because the input box re-renders due to some loading state changes.
```
[see this comment](https://github.com/elastic/kibana/pull/152962#issuecomment-1462339962).

Reply:
Thank you @e40pud for your review and for explaining why it is skipped 🙏🏻, I will give it a second try if didn't work then I will create a ticket for it

Reply:
Skipping flyout_validation until we fix the `exception entry flyout` multiple rendering issue